ci/test-container.sh

```diff
@@ -68,4 +74,17 @@ if [[ $n_downloaded != 1 ]]; then
   fatal "Expected 1 'Downloading', but got $n_downloaded"
 fi
 
+# test kernel replacement
+kernel_suffix=${kver}/${krev}.fc${versionid}/x86_64/kernel-${kver}-${krev}.fc${versionid}.x86_64.rpm
+modules_suffix=${kver}/${krev}.fc${versionid}/x86_64/kernel-modules-${kver}-${krev}.fc${versionid}.x86_64.rpm
+core_suffix=${kver}/${krev}.fc${versionid}/x86_64/kernel-core-${kver}-${krev}.fc${versionid}.x86_64.rpm
+
+KERNEL_URL=https://kojipkgs.fedoraproject.org//packages/kernel/$kernel_suffix
+MODULES_URL=https://kojipkgs.fedoraproject.org//packages/kernel/$modules_suffix
+CORE_URL=https://kojipkgs.fedoraproject.org//packages/kernel/$core_suffix
```

Comment:
Minor: could also do something similar to the Ignition test and just paste the koji URL.